src/Plugin.php

```diff
@@ -86,6 +90,45 @@ public static function upload_mime_types(array $mimes) {
   }
 
   /**
+   * Removes svg width and height attributes from the content.
+   *
+   * @implements the_content
+   */
+  public static function the_content($content) {
+    $content = preg_replace_callback('@<img [^>]*src=.+?.svg[^>]*>@', function ($matches) {
```

Comment:
Because `.+?.svg` causes it to also match this (entire string):

```
<img src="foo.png">whatever<img src="bar.svg">
```
---
Lastly, I also just noticed that you did not escape the dot before the `svg` file extension, so it actually matches any character instead of only a period.